test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -38,7 +38,7 @@ String getConnectionString(int port) {
     }
 
     @Test
-    public void AddNodeTest() throws Exception {
+    public void AddRemoveNodeTest() throws Exception {
```

Comment:
This should have a comment as to what is being tested

Reply:
Also I think we need to run some combination of these actions in multiple tests.
Like:
1. add node -> remove node -> add node -> add node
2. add node -> add node -> remove node -> add node
3. add node -> add node -> add  node -> remove node -> remove node -> add node -> add node
and so on. 
I understand these are slow tests but we need to catch as many issues as we can as early as we can.

Reply:
I can't test these scenarios in this PR because I need force remove. I will add these test cases to the force remove PR.